modules/installing-ocp-agent-ibm-z-zvm.adoc

```diff
@@ -55,4 +56,4 @@ You can use the `CP PUNCH` command or, if you use Linux, the `vmur` command, to
 $ ipl c
 ----
 +
-For more information, see link:https://www.ibm.com/docs/en/zvm/latest?topic=commands-ipl[IPL] in IBM Documentation.
\ No newline at end of file
+For more information, see link:https://www.ibm.com/docs/en/zvm/latest?topic=commands-ipl[IPL] (IBM Documentation).
```

Comment:
Should a link be provided as technically IBM Is not a third party vendor.

Reply:
I think so, it's external to our docs, so it's probably useful to just link it for users. Either way, I was not the one who originally added the link (and this overall module) so I do not want to be overly intrusive to this doc aside from making some simple style guide changes